webpack.config.js

```diff
@@ -3,7 +3,16 @@
 const { fork } = require('child_process');
 const webpack = require('webpack');
 const path = require('path');
+const { createHash } = require('crypto');
+const CleanCSS = require('clean-css');
```

Comment:
We don't appear to have a dependency on this 🤔. If it currently exists only as an indirect dependency can we make our requirement for it explicit by adding it as a direct dependency?

Reply:
yes 👍 was planning to but forgot!